src/controllers/abstract-controller.ts

```diff
@@ -9,6 +9,11 @@ interface ParsedErrors {
   [key: string]: string[];
 }
 
+interface ConstructorFabric {
+  create(...args: any): any;
```

Comment:
Maybe we could add generic type here instead of `any`. But its only a guess.

Reply:
done! With few notes:

This interface is used locally only. In short, it is not reused. So I just defined the return values more strictly. But overall, it's a bit useless. Nobody will write a constructor that doesn't return an object unless it has a special intention to do so.

Also, it would be more important to show that a class with validated fields is passed here. It would be really useful. But I doubt it's possible.

@SerafimovichEugene